src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -95,5 +95,21 @@
   <ItemGroup>
     <None Include="$(OutputPath)/**/$(AssemblyName).pdb" Pack="true" PackagePath="lib" Visible="false" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="CallbackConfigurationExtensions.Generated.tt">

```

Comment:
Having the `Generated` as part of the filename seems odd to me, given that this _isn't_ a generated file, but I trust your knowledge of the conventions.

Reply:
I know, I'm not too happy about it either. But there's no easy way to control the output file name, it's always based on the template file name. Maybe I could name it CallbackConfigurationExtensions.StronglyTyped.tt or something similar ?